app/seed_games/smash64.rb

```diff
@@ -0,0 +1,52 @@
+class Smash64
+  def info 
+      {
```

Comment:
This is a long winded way to seed data. its just copied data from the API. apparently we can use a PG_DUMP instead

Reply:
Agreed, I'd be interested in learning more about PG_DUMP as well